### PR TITLE
Write class diagram

### PR DESCRIPTION
**Pull request by nefario19**
We've added the class diagram. Let's merge this.

### CODE REVIEW

**Comment by nefario19** [comment]
Added enum in the class diagram last minute. Please review those. See the last commit.

File: app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml [hunk 1]
```diff
@@ -0,0 +1,102 @@
+@startuml
+class News {
+    -title : String
+    -body : String
+}
+class Website {
+' Attributes
```
Comment (samjakbal) [inline app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml:7]: We should decide to either have these comments at every class definition or remove them entirely. I'm fine either way as long as it is consistent.

File: app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+}
+class Activity {
+    -title : String
+    -description : String
+    -startDate : Date
+    -endDate : Date
+    -startTime : Time
+    -endTime : Time
+}
+'class Car {
```
Comment (samjakbal) [inline app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml:67]: This class is no longer used. It should be removed.

File: app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+User "1" -up-> "1" MemberList : views >
+User "1" -up-> "1" Activity : views >
+User "1" -up-> "0..*" Achievement : wins >
+User "1" -up-> "0..*" News : reads >
+Website "1" --> "0..*" WebPage : scrapes >
+NewsWebPage "1" --> "0..*" News : generates >
+MemberListWebPage "1" --> "0..*" MemberList : generates >
+MemberListWebPage "1" --> "0..*" Achievement : generates >
+ActivityWebPage "1" --> "0..*" Activity : generates >
+'User "1" --> "1" Car: Offers >
```
Comment (samjakbal) [inline app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml:101]: This line should be removed as the `Car` class is no longer used.

File: app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml (hunk 1 above)
Comment (nefario19) [reply app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml]: Solved this with the latest commit.

File: app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml (hunk 2 above)
Comment (nefario19) [reply app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml]: Solved this with the latest commit.

File: app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml (hunk 3 above)
Comment (nefario19) [reply app/src/main/java/com/stamsa/yushinkan/class_diagram_yushinkan.puml]: Solved this with the latest commit.

**Review by samjakbal — APPROVED** [review]

**Pull request merged**